<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>build</class>
 <widget class="QWidget" name="build">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>354</width>
    <height>169</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="KTabWidget" name="ktabwidget">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="errs">
      <attribute name="title">
       <string>Errors &amp;&amp; Warnings</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTreeWidget" name="errTreeWidget">
         <property name="autoScroll">
          <bool>false</bool>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="allColumnsShowFocus">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string comment="Header for the file name column">File</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string comment="Header for the line number column">Line</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string comment="Header for the error message column">Message</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="output">
      <attribute name="title">
       <string>Output</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPlainTextEdit" name="plainTextEdit">
         <property name="tabChangesFocus">
          <bool>true</bool>
         </property>
         <property name="undoRedoEnabled">
          <bool>false</bool>
         </property>
         <property name="lineWrapMode">
          <enum>QPlainTextEdit::WidgetWidth</enum>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="settings">
      <attribute name="title">
       <string>Settings</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Working directory:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>buildDir</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Build command:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>buildCmd</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="3">
        <widget class="KLineEdit" name="buildCmd">
         <property name="showClearButton" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Clean command:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>cleanCmd</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="3">
        <widget class="KLineEdit" name="cleanCmd">
         <property name="showClearButton" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Quick compile:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>quickComp</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="3">
        <widget class="KLineEdit" name="quickComp">
         <property name="toolTip">
          <string>Use &quot;%f&quot; for current file and &quot;%d&quot; for directory of current file.</string>
         </property>
         <property name="showClearButton" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>178</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <widget class="KLineEdit" name="buildDir">
         <property name="toolTip">
          <string>Leave empty to use the directory of the current document. </string>
         </property>
         <property name="showClearButton" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QToolButton" name="browse">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KTabWidget</class>
   <extends>QTabWidget</extends>
   <header>ktabwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>buildDir</tabstop>
  <tabstop>buildCmd</tabstop>
  <tabstop>cleanCmd</tabstop>
  <tabstop>quickComp</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
